Libraries/LibC/assert.h

```diff
@@ -41,7 +41,7 @@ __attribute__((noreturn)) void __assertion_failed(const char* msg);
         } while (0)
 #    define ASSERT_NOT_REACHED() assert(false)
 #else
-#    define assert(expr) ((void)0)
+#    define assert(expr) (void(0))
```

Comment:
Is `void(0)` even correct C?

Reply:
@bugaevc both are valid in C++, but I believe that `void(0)` is **not** valid C. Basically it is using C++'s "constructor like cast syntax". Just as you can do `(int)5.2` you can also do `int(5.2)`.